docs/apm/correlations.asciidoc

```diff
@@ -0,0 +1,91 @@
+[role="xpack"]
+[[correlations]]
+=== Find latency and error correlations
+
+**Correlations** surface attributes of your data that are potentially correlated with high-latency or erroneous transactions.
+Surfaced attributes are user-defined, meaning that they are completely customizable to your APM data.
+Find something interesting? A quick click of a button will auto-query your data as you work to resolve the underlying issue.
+
+For example, a site reliability engineer, who is responsible for keeping production systems up and running,
+notices an increase in latency in certain transactions.
+Analyzing metadata or tags that exist in high-latency transactions but not in lower-latency transactions
+can potentially point towards the root cause.
+They may find that a particular piece of hardware, like a host or pod, has failed, increasing latency.
+Or, perhaps a set of users, based on IP address or region, is physically too far away from the nearest
+data center, increasing latency.
+
+[discrete]
+[[view-correlations]]
+===  View correlations
+
+With a service selected, click **View correlations**:
+
+[role="screenshot"]
+image::apm/images/correlations.png[Correlations]
```

Comment:
@bmorelli25 I noticed for all the screenshots that we're using the v8 theme - was this decided within the docs team that any new screenshots would use the "not yet" released theme? Just wondering since I don't think we're shipping the theme publically until a few minors.

Reply:
🤦 This is an oversight on my part